tools/scripts-utils/src/path.js

```diff
@@ -23,22 +23,10 @@ function resolveBin(modName, { executable = modName, cwd = process.cwd() } = {})
 	} catch (_error) {
 		// ignore _error
 	}
-	try {
-		const modPkgPath = require.resolve(`${modName}/package.json`);
-		const modPkgDir = path.dirname(modPkgPath);
-		const mod = JSON.parse(fs.readFileSync(modPkgPath));
-		const { bin } = mod.bin;
-		const binPath = typeof bin === 'string' ? bin : bin[executable];
-		const fullPathToBin = path.join(modPkgDir, binPath);
-		if (fullPathToBin === systemCommandPath) {
-			return executable;
-		}
-		return fullPathToBin.replace(cwd, '.');
```

Comment:
this code is supposed to look into package.json bin attribute but nodejs is creating a .CMD file for that on win32 so the path will never been equal and the output will never be useful